infrastructure/projects.yml

```diff
@@ -4,6 +4,10 @@ projects:
     cocina_level2: false
   - repo: sul-dlss/assembly-objectfile
     cocina_level2: false
+  - repo: sul-dlss/assembly-objectfile
+    cocina_level2: false
+  - repo: sul-dlss/blacklight-hierarchy
+    cocina_level2: false
```

Comment:
This gem is used more by Access than us. I wonder why we'd want to start maintaining that gem.